src/app/_components/simple-error-page.tsx

```diff
@@ -0,0 +1,4 @@
+export function SimpleErrorPage(props: { error: string }) {
```

Comment:
Add TODO to use this more.

Reply:
Added a ticket - https://www.notion.so/Review-where-error-is-received-and-see-if-SimpleErrorPage-can-be-used-323bba93688742419a78f5598e898e08?pvs=4